specs/protocol/predeploys.md

```diff
@@ -347,6 +347,17 @@ Address: `0x4200000000000000000000000000000000000021`
 
 The `EAS` predeploy implements the `Ethereum Attestation Service` protocol.
 
+## RevenueSharer
+[Implementation](https://github.com/ethereum-optimism/optimism/blob/6e12d4b630cc1d3dbhttps://github.com/ethereum-optimism/optimism/blob/gk/revenue-sharing/packages/contracts-bedrock/src/L2/RevenueSharer.sol) 
+ <!---
+TODO: update above link when implementation is merged
+-->
+
+Address: `0x4200000000000000000000000000000000000022`
```

Comment:
This collides with an interop defined address: https://specs.optimism.io/interop/predeploys.html#crossl2inbox

Reply:
So it does. I missed that one since it wasn't listed in [this table.](https://specs.optimism.io/protocol/predeploys.html#overview). 

Update the `RevenueSharer` here [19647e6](https://github.com/ethereum-optimism/specs/pull/115/commits/19647e6473e207167fa5d26e3ea69081263c4bcb).